package.json

```diff
@@ -35,7 +35,7 @@
     "material-ui-search-bar": "1.0.0-beta.13",
     "md5": "2.2.1",
     "moment": "2.24.0",
-    "npm": "6.9.0",
+    "npm": "^6.13.1",
```

Comment:
Please remove dependency of npm from `package.json`. It shouldn't be there.

Reply:
Yes, i have removed dependancy of npm from package.json